### PR TITLE
Feature/dx 1130 trustly corrections

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002111&bid=19677680)

File: payments/trustly/redirect.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Content-Type: application/json
         "language": "sv-SE",
         "urls": {
             "completeUrl": "https://example.com/payment-completed",
-            "cancelUrl": "https://example.com/payment-canceled",
+            "hostUrl": "https://example.com",
```
Comment (asbjornu) [inline payments/trustly/redirect.md:69]: I believe this field is called `hostUrls` and is an array everywhere else?

```suggestion
            "hostUrls": [ "https://example.com" ],
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002111&bid=19677680)

**Review by asbjornu — APPROVED** [review]

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6002111&bid=19677680)

**Pull request merged**